components/x-teaser-timeline/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@financial-times/x-teaser-timeline",
+  "version": "0.0.0",
+  "description": "",
+  "main": "dist/TeaserTimeline.cjs.js",
+  "module": "dist/TeaserTimeline.esm.js",
+  "browser": "dist/TeaserTimeline.es5.js",
+  "style": "dist/TeaserTimeline.css",
+  "scripts": {
+    "prepare": "bower install && npm run build",
+    "build": "node rollup.js",
+    "start": "node rollup.js --watch"
+  },
+  "keywords": [
+    "x-dash"
+  ],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@financial-times/x-article-save-button": "0.0.11",
+    "@financial-times/x-engine": "file:../../packages/x-engine",
+    "@financial-times/x-teaser": "file:../x-teaser",
+    "classnames": "^2.2.6",
```

Comment:
`classnames` or `classcat` @quarterto ? I think we might prefer the latter.

Reply:
we are already using classnames in x-styling-demo

classcat is about 70b smaller
https://bundlephobia.com/result?p=classnames@2.2.6
https://bundlephobia.com/result?p=classcat@3.2.5

Reply:
i had run into bundling problems with `classnames` but this might have been fixed by updates to it or our tooling 🤷‍♀️